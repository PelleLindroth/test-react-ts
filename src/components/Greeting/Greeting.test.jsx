import { render, screen } from '@testing-library/react'
import { shallow } from "enzyme"
import Greeting from './Greeting'

describe('Greeting', () => {
 it('renders h1', () => {
  render(<Greeting />)

  const hello = screen.getByText(/hello/i)

  expect(hello).toBeInTheDocument()
 })
 it('shallow renders', () => {
   const wrapper = shallow(<Greeting />)

   expect(wrapper).toMatchSnapshot()
 })
})