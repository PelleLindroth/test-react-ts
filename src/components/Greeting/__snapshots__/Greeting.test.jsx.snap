// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting shallow renders 1`] = `
<h1>
  Hello world again
</h1>
`;
